backend/src/controllers/userControllers.ts

```diff
@@ -0,0 +1,190 @@
+import { NextFunction, Request, Response } from "express";
+import { STATUS, STATUS_MESSAGE } from "../consts/status-codes";
+import User from "../models/userModel";
+import logger from "../logger/logger";
+import { PAGINATION } from "../consts/pagination";
+
+const createNewUser = async (
```

Comment:
Ensure password is hashed before saving the user to the database. Add input validation to ensure data integrity.